rust/common-runtime/src/policy.rs

```diff
@@ -0,0 +1,109 @@
+use crate::{CommonRuntimeError, InputOutput};
+use common_ifc::{Context as IfcContext, Policy};
+
+/// A type that wraps an inner value that has been validated against some [Policy].
```

Comment:
What is the lifecycle of these `Validated` wrappers? To validate node input every time before executing, or once at validation time? Or maybe not yet relevant without full graphs

Reply:
The later. "To be defined." The only thing I wanted to do here was make it explicit that validation happened outside of the steps to invoke with some inputs.